components/cas-inbound-authenticator/src/main/java/org/wso2/carbon/identity/sso/cas/processor/SSOLoginProcessor.java

```diff
@@ -76,7 +73,31 @@ public boolean canHandle(IdentityRequest identityRequest) {
         CASResponse.CASResponseBuilder builder = new CASLoginResponse.CASLoginResponseBuilder(casMessageContext);
         String serviceUrlFromRequest = casMessageContext.getServiceURL();
         AuthenticationResult authnResult = processResponseFromFrameworkLogin(casMessageContext, identityRequest);
-        String acsURL = CASSSOUtil.getAcsUrl(serviceUrlFromRequest, casMessageContext.getRequest().getTenantDomain());
+        URL url = null;
+
+        try {
+            url = new URL(serviceUrlFromRequest);
+        } catch (MalformedURLException mfe) {
+            throw new FrameworkException("Error occurred while retrieving cas service url from: " + serviceUrlFromRequest, mfe);
```

Comment:
In the wso2 we avoid lines longer than 120 characters since they’re not handled well by many terminals and tools. Please follow this documentation https://github.com/wso2/product-is/wiki/Contributing-to-Identity-Server for further reference